components/Footer.js

```diff
@@ -10,19 +10,19 @@ function Footer() {
         <div className="footer d-flex justify-content-lg-around justify-content-center align-items-center flex-column flex-md-row p-2 py-md-5">
             <div className="py-2 my-md-0">
                 <h2>Follow FreeFrom</h2>
-                <a>
+                <a href="https://www.facebook.com/freefrom.org" target="_blank" rel="noreferrer">
```

Comment:
Is there a reason you didn't add `noopener` to the `rel` attribute on these links? As far as I know, best practice security-wise is to use `rel="noreferrer noopener"`, but I don't understand much more beyond that.

Reply:
@egiurleo Ooh I wasn't aware, I only added `noreferrer` because ESLint yelled at me :) 

Just researched it:

> The noopener is needed to enhance the security of your website and prevent other websites from gaining access to your page (through the browser session). 
The noreferrer is used to protect referral information from being passed to the target website and this also hides referral traffic in Google analytics.

Seems harmless so I'll go ahead and add `noopener` as well!